static/scss/add-link-menu.scss

```diff
@@ -0,0 +1,19 @@
+.add-link-menu {
```

Comment:
Shouldn't we add this to `editor.scss` instead of creating a new file?

Reply:
I've been preferring to keep CSS specific to a particular component in separate files recently, I think it prevents CSS files from getting too big (which is a big hindrance to refactoring).

Reply:
In this case, the menu is only ever relevant to the editor component. I agree that bloated CSS files are a problem broadly speaking, but I would argue that (a) the menu that is being styled is only ever relevant to the editor, (b) it's a different form of bloat to have a ton of separate files for somewhat limited definitions, and (c) we may end up finding that it's not a good pattern to couple our approach to separating CSS files with the concept of React components. I don't feel too strongly about it, so if you prefer to leave as is, no worries